packages/engine-server/src/markdown/remark/dendronPub.ts

```diff
@@ -152,7 +151,19 @@ function plugin(this: Unified.Processor, opts?: PluginOpts): Transformer {
           )}`,
         });
       }
-      const note = MDUtilsV5.getNoteByFname(proc, { fname });
+      let note;
+
+      // Special Logic for 403 Error Static Page:
+      if (fname === "403") {
+        note = SiteUtils.addSiteOnlyNotes({ engine })[0];
```

Comment:
can we refactor addSiteOnlyNotes since this is fairly brittle? eg. `SiteUtils.create403Note` and use this instead